### ISSUE TITLE
It's broken

### ISSUE THREAD

**Issue by Josh-Myers** [simon--poole/ev-browser-extension-chrome#3]
My chrome event vods no longer hides the video length. The youtube ads seem different not sure if that has changed something.

**Comment by simon--poole** [comment]
Investigating this now - thanks for letting me know.

**Comment by Josh-Myers** [comment]
I saw some comment now saying it is related to the channel not being added: 

"Not all of the channels these vods are on are included in the spoiler prevention plugin (which is incredible by the way). Not the end of the world, just a heads up."
https://www.reddit.com/r/DotaVods/comments/4gyasa/weplay_dota2_league_season_3/

I tried one of the videos for the older events and it worked for me so that must be my problem. 
Thanks for this awesome extension.
